### ISSUE TITLE
iOS 10 , how many notification can be schedule?

### ISSUE THREAD

**Issue by shivam5x** [katzer/cordova-plugin-local-notifications#1123]
Hi,
 I got new updated plugin cordova plugin add https://github.com/katzer/cordova-plugin-local-notifications#ios10 which supprot in ios 10 for both foreground  and background .
But when i check my app in ios 9 it is scheduling as i setup more than one notification, but in ios 10 , it doesn't notify me something.
Please help us out .

Thanks

**Comment by WilliamALiang** [comment]
@shivam5x Hi:Your problem solved?

**Comment by shivam5x** [comment]
Not exactly but I have debugged and found something behavior difference between ios9 and ios 10 plugin. 

In ios9 ,plugin schedule notification randomly like multiple thread execute single method , i.e. if I set loop for 1 to 30 then no order will be follow to schedule notification or sometime it will ignore to schedule like 2,4,7 (id of notification). 

In ios10, behavior is opposite of ios9 plugin , i.e. it will schedule notification sequentially 1,2,3   till 64 which is limit of ios notification.
